turbo/jsonrpc/send_transaction.go

```diff
@@ -68,7 +68,7 @@ func (api *APIImpl) SendRawTransaction(ctx context.Context, encodedTx hexutility
 	api.SenderLocks.AddLock(sender)
 	defer api.SenderLocks.ReleaseLock(sender)
 
-	if txn.Type() != types.LegacyTxType {
+	if txn.Type() != types.LegacyTxType && !api.UsingEthereumHardfork {
```

Comment:
Do we need this?  If we're going to support London from the get-go on a network we can just adjust the chainspec to allow this by setting the London height to 0 rather than the 999...999 value we use in ZK configs at the moment.